src/main/java/me/kavin/piped/Main.java

```diff
@@ -38,7 +40,7 @@ public static void main(String[] args) throws Exception {
 		QueryStringDecoder query = new QueryStringDecoder(req.uri());
 
 		try {
-		    return writeResponse(res, query.parameters().get("hub.challenge").get(0), 200, "private", start);
+		    return writeResponse(res, query.parameters().get("hub.challenge").get(0), APPLICATION_XML, 200, "private", start);

```

Comment:
This should be `TEXT_PLAIN`

Reply:
This is because the query parameter is just a random number/string.

Reply:
```suggestion
		    return writeResponse(res, query.parameters().get("hub.challenge").get(0), TEXT_PLAIN, 200, "private", start);

```

Reply:
Please next time use diff, it's better to preview and we can commit if the change is small (like this one)